visualizer/IdealGraphVisualizer/pom.xml

```diff
@@ -117,7 +117,7 @@
         <module>ViewerApi</module>
     </modules>
     <properties>
-        <netbeans.version>RELEASE220</netbeans.version>
+        <netbeans.version>RELEASE260</netbeans.version>
```

Comment:
With this change I can built regularly:
```bash
/graal/visualizer/IdealGraphVisualizer$ JAVA_HOME=/graalvm-21/ mvn install
```
and then execute on GraalVM 24.0.1 (which was failing previously):
```bash
/graal/visualizer/IdealGraphVisualizer$ JAVA_HOME=/graalvm-24 mvn -f application/ -Dexec.vmArgs= -Dexec.appArgs= --no-transfer-progress nbm:cluster-app nbm:run-platform
```